andy/src/test/java/unit/writer/weblab/WebLabResultWriterTest.java

```diff
@@ -19,7 +19,7 @@
 import static org.assertj.core.api.Assertions.allOf;
 import static org.assertj.core.api.Assertions.assertThat;
 import static unit.writer.standard.StandardResultTestAssertions.*;
-import static unit.writer.weblab.WebLabHighlightsJsonTestAssertions.*;
+import static unit.writer.weblab.WebLabEditorFeedbackJsonTestAssertions.*;
 
 public class WebLabResultWriterTest extends StandardResultWriterTest {
```

Comment:
Do we have a test for when there's no thing to show in the highlights?

Reply:
This test class extends `StandardResultWriterTest`, so all tests for the standard result writer are also executed for the WebLabResultWriter. The tests there don't have any highlights set.

The tests there implicitly verify that the WebLabResultWriter doesn't crash when there are no highlights. But we don't have any assertions to check that nothing is written to the highlights file.

Reply:
Ah, great! 

Is this assertion needed?

Reply:
I don't think so. I don't think there's any reasonable bug that could occur which would cause false data to be generated and written to the file if no such data is provided to the writer. And we already have tests that check that the writer doesn't throw an exception in this case.

Reply:
In fact, we also have system tests which would not generate any highlights. They check only the score and implicitly verify that Andy doesn't crash in this case.